diff --git a/engine/src/nnue.h b/engine/src/nnue.h
index d0ed5bc..326bdaa 100644
--- a/engine/src/nnue.h
+++ b/engine/src/nnue.h
@@ -146,7 +146,7 @@ INCBIN(nnue3, "nets/feanor.nnue");
       reinterpret_cast<const NNUE_Params *>(g_nnue2Data),
       reinterpret_cast<const NNUE_Params *>(g_nnue3Data),
   };
   PATRICIA_NNUE_ASSERT(index < nets.size());
-  return *std::assume_aligned<64>(nets[index]);
+  return *reinterpret_cast<const NNUE_Params *>(__builtin_assume_aligned(nets[index], 64));
 }
 
 [[nodiscard]] inline constexpr size_t phase_to_index(int phase) noexcept {
